weasyprint/css/utils.py

```diff
@@ -699,6 +699,11 @@ def get_content_list_token(token, base_url):
     if string is not None:
         return string
 
+    # <var>
```

Comment:
Here a variable function in a content declaration is acknowledged and returned accordingly.